tools/function_finder/function_finder_saturn.py

```diff
@@ -176,22 +105,21 @@ def print_github_flavored_table(data):
             results = [f.result() for f in futures]
 
     to_print = []
-
     for i, o in enumerate(output):
-        obj = {
-            "Filename": str(o["filename"]).replace("asm/saturn/", ""),
-            "Function Name": o["function_name"],
-            "Length": o["length"],
-            "Branches": o["branches"],
-        }
+        row = [
+            str(o["filename"]).replace("asm/saturn/", ""),
+            o["function_name"],
+            o["length"],
+            o["branches"],
+            "",
+            "",
+        ]
 
         if len(results) and results[i]:
-            obj["WIP"] = results[i]["link"]
-            obj["%"] = results[i]["percent"]
-        else:
-            obj["WIP"] = ""
-            obj["%"] = ""
+            row[4] = results[i]["link"]
+            row[5] = results[i]["percent"]
 
-        to_print.append(obj)
+        to_print.append(row)
 
-    print_github_flavored_table(to_print)
+    headers = ["Filename", "Function Name", "Length", "Branches", "WIP", "%"]
+    print(tabulate(to_print, headers=headers, tablefmt="github"))
```

Comment:
I thought I tried this exact thing and couldn't get it to display right on GitHub. Do you have a branch pushed with the result?

Reply:
It seems to work just fine. I originally tested it with a subset. This is the full dump after running the tool from the latest `make extract_saturn`: https://gist.github.com/Xeeynamo/a41713c9b2a610a67c95199641a19394